<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="computeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportedPathButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exportedPathEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>521</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/pariterre/programmation/QtKinoveaAnalyse/box_jump_tp.xml</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fichier export√© de Kinovea :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fichier de configuration des segments :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exportedConfigEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>521</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/pariterre/programmation/QtKinoveaAnalyse/Winter.xml</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportedConfigButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
